src/UnscentedKalmanInversion.jl

```diff
@@ -7,23 +7,23 @@ An unscented Kalman Inversion process.
 """
 mutable struct Unscented{FT <: AbstractFloat, IT <: Int} <: Process
     "a vector of arrays of size `N_parameters` containing the mean of the parameters (in each `uki` iteration a new array of mean is added)"
-    u_mean::Vector{Vector{FT}}
+    u_mean::AbstractVector{Vector{FT}}
```

Comment:
I would leave this field untyped as you just need an iterable of some vector type